src/Lifecycle.ts

```diff
@@ -835,7 +831,17 @@ async function doSetLoggedIn(
         await abortLogin();
     }
 
-    const tokenRefresher = await createOidcTokenRefresher(credentials);
+    let storedClientid;
+    try {
+        storedClientid = getStoredOidcClientId();
+    } catch {}
```

Comment:
Seems odd to do this twice, why not just pass it into `createOidcTokenRefresher`